test/unit_tests/opencl/unit_test_helper_opencl.cpp

```diff
@@ -13,7 +13,7 @@ namespace
     const std::string programCode = R"(
         #pragma OPENCL EXTENSION cl_khr_global_int32_base_atomics : enable
 
-        __kernel void isEqualOpenCL( __global const uchar * data, uchar value, uint rowSize, uint width, uint height, volatile __global uint * differenceCount )
+        __kernel void isEqualOpenCL( __global const uchar * data, uint offset, uchar value, uint rowSize, uint width, uint height, volatile __global uint * differenceCount )
```

Comment:
We added offset variable which is not used anywhere. We should modify line `const size_t id = y * rowSize + x;` into `const size_t id = offset + y * rowSize + x;` for both functions.

Reply:
Noted on this and applied the change.